test/unit/lib/storages/RedisProxyCache.test.ts

```diff
@@ -27,12 +27,12 @@ import { IoRedisMock } from '../../mocks';
 jest.mock('ioredis', () => IoRedisMock);
 
 import { setTimeout as sleep } from 'node:timers/promises';
-import { createProxyCache, IProxyCache, STORAGE_TYPES } from '#src/index';
+import { createProxyCache, IProxyCache, STORAGE_TYPES } from '../../../../src/index';
```

Comment:
why do we need to change  `#src/index` to `../../../../src/index`?

Reply:
@geka-evk The `#` format breaks code navigation, at least in VS Code in my environment.

Reply:
so it "breaks" navigation only in one place, but not in others? And what does it mean - "breaks"?

![image](https://github.com/user-attachments/assets/080ac824-e0b2-4877-ba6f-f88283e51456)

Reply:
<img width="824" alt="Screenshot 2024-08-19 at 09 16 14" src="https://github.com/user-attachments/assets/20c8a0aa-2867-413f-b546-41fdf5310f78">